core/src/main/scala/org/apache/spark/SparkSnappyUtils.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 /*
- * Copyright (c) 2018 SnappyData, Inc. All rights reserved.
+ * Copyright (c) 2017-2019 TIBCO Software Inc. All rights reserved.
```

Comment:
One query for my understanding- Do we need to change copyright notice for all files being modified henceforth?

Reply:
There is some discussion going on. All files will be changed in one go at some point so nothing explicit to be done right now.

Reply:
Ok, thanks!